tests/test_filter_panels.py

```diff
@@ -0,0 +1,91 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_edit_filter_panels(base_url, selenium):
+    """Open Treeherder page, open Filters Panel, select one filter,
+    verify results"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    # Get next 50 jobs to see 'exception' unclassified failures
+    page.get_next_fifty_results()
+    page.deselect_all_failures()
+
+    # Check 'testfailed' unclassified failures
+    page.select_testfailed_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['testfailed']
+    page.select_testfailed_failures()
```

Comment:
This seems like an unnecessary or duplicate step after you've made the assert, and could be removed.